azkaban-web-server/build.gradle

```diff
@@ -117,7 +117,7 @@ task jsTest(dependsOn: ['npm_install'], type: NodeTask) {
     script = file('node_modules/mocha/bin/mocha')
 
     // args keep the test files' location
-    args = ['src/web/js/azkaban/test/']
+    args = ['src/web/js/azkaban/test/test.js']
```

Comment:
Does this change mean "run only the old test under this task"?

Reply:
Correct.  This is to prevent `gradle test` from running everything in the test directory.   Maybe it's better to reorg the directory tree?  

```
src/web/js/azkaban/test/
                                       ├── test_running
                                       └── test_static
```

This would allow us to keep the current `args` behavior of running all tests in the test directory, but still have separation between gradle & NPM tests.

Reply:
I like it. Or maybe just `src/web/js/azkaban/test_running` and `src/web/js/azkaban/test_static`, no need for (empty?) `src/web/js/azkaban/test`?

Reply:
Sounds good.  I'll submit a updated PR for the directory restructuring and remove the `test` dir.